lib/hocon/impl/config_delayed_merge.rb

```diff
@@ -56,4 +56,79 @@ def hash
     # note that "origin" is deliberately NOT part of equality
     @stack.hash
   end
+
+  def render_to_sb(sb, indent, at_root, at_key, options)
```

Comment:
My brain hurts remembering doing this, but I think that I used `render_to_sb` because I needed the `at_key` parameter to call `render_value_to_sb_from_stack`...